pandas/core/generic.py

```diff
@@ -5716,24 +5716,81 @@ def clip_upper(self, threshold, axis=None, inplace=False):
 
     def clip_lower(self, threshold, axis=None, inplace=False):
         """
-        Return copy of the input with values below given value(s) truncated.
+        Return copy of the input with values below given value(s) trimmed.
+
+        If an element value is below the threshold, the threshold value is
```

Comment:
Since threshold is an object here you can just say "value is below \`threshold\`..." (note backticks)

Reply:
Also changing the wording a little to match #20212 as per @datapythonista suggestion.